ci-operator/step-registry/rosa/cluster/upgrade/hcp/rosa-cluster-upgrade-hcp-commands.sh

```
@@ -251,10 +225,7 @@ while true; do
   # Upgrade cluster when there is upgrade path,otherwise return log 
   if [ -n "$available_version" ]; then
     upgrade_cluster_to $available_version
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)